src/axom/primal/operators/closest_point.hpp

```diff
@@ -225,6 +226,27 @@ inline Point<T, NDIMS> closest_point(const Point<T, NDIMS>& pt,
   return Point<T, NDIMS>(res.array());
 }
 
+/*!
+ * \brief Computes the closest point from a point, P, to a sphere.
+ *
+ * \param [in] P the query point
+ * \param [in] sphere user-supplied triangle
+ * \return cp the closest point on sphere \a sphere to point P
+ *
+ * \note The closest point is uniquely defined everyone except at the sphere's origin.
+ * We handle that case by returning the point on the sphere along the direction
+ * determined by \a primal::Vector::unitVector() for the zero vector
+ */
+template <typename T, int NDIMS>
+AXOM_HOST_DEVICE inline Point<T, NDIMS> closest_point(const Point<T, NDIMS>& P,
```

Comment:
New function to find the closest point on a sphere to a point in space.
This was in support of testing the new `SignedDistance` functionality.

Reply:
Should documentation for parameter sphere be sphere instead of triangle?

Reply:
I think @kennyweiss is referring to a sphere described as a triangulated surface mesh, but it's not clear.

Reply:
>  * \param [in] sphere user-supplied triangle

Thanks @gunney1 -- good catch -- it was a copy/paste error. I'll fix it.

Reply:
fixed